lib/templates/elements/futurism_utils.js

```diff
@@ -0,0 +1,28 @@
+/* global IntersectionObserver, CustomEvent, setTimeout */
+
+const dispatchAppearEvent = function (that, entry, observer) {
```

Comment:
there's a best practice here I'd like you to follow.

similar as in stimulus_reflex, write a function that dynamically assigns (mixes in) methods into an object:

```js
const extendElement = element => {
  Object.assign(element, {
    connectedCallback () {
       // ...
    },
    dispatchAppearEvent( ...) {
      // ...
   })
}
```

in the element's constructor you can then do

```js
constructor() {
   super()
   extendElement(this)
}
```

that way, you save the `that` shenanigans

Reply:
For reference, take a look here: https://github.com/hopsoft/stimulus_reflex/blob/master/javascript/stimulus_reflex.js#L69

or here:
https://github.com/stimulus-use/stimulus-use/blob/master/src/use-intersection/use-intersection.ts#L34